composer.json

```
@@ -12,7 +12,7 @@
         "symplify/rule-doc-generator-contracts": "^11.1",
         "nette/utils": "^3.2|^4.0",
         "sebastian/diff": "^5.1|^6.0",
-        "illuminate/container": "^10.39",
+        "illuminate/container": "^11.0",
```

```suggestion
        "illuminate/container": "^10.39|^11.0",
```

Why break backwards compatibility?

---

Yeah, it's better.